coordinator/coordinator.go

```diff
@@ -198,9 +198,10 @@ func (k *KubernetesJobSpawner) GetJobRunner(jobName string, config runner.Config
 			"ETCD_CONFIG":      string(serializedETCD),
 			"K8S_RUNNER_IMAGE": pandas_image,
 		},
-		Image:    help.GetEnv("WORKER_IMAGE", "local/worker:stable"),
-		NumTasks: 1,
-		Resource: resourceId,
+		JobPrefix: "runner",
+		Image:     help.GetEnv("WORKER_IMAGE", "local/worker:stable"),
```

Comment:
+1

Reply:
a little confused why we would add this to the config?

Reply:
we use it in config to be consistent with the other stuff as well as in case we want to change the default value, we can change it in one place if/when we use this in multiple locations.